resolver/contentResolver.go

```diff
@@ -0,0 +1,70 @@
+package resolver
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/Financial-Times/transactionid-utils-go"
+	"io/ioutil"
+	"net/http"
+)
+
+type ContentResolver interface {
+	ResolveContents(uuids []string, tid string) ([]map[string]interface{}, error)
+}
+
+type defaultContentResolver struct {
+	contentResolverAppURI string
+	httpClient            *http.Client
+}
+
+func NewContentResolver(client *http.Client, contentResolverAppURI string) ContentResolver {
+	return &defaultContentResolver{contentResolverAppURI: contentResolverAppURI, httpClient: client}
+}
+
+func (cr *defaultContentResolver) ResolveContents(uuids []string, tid string) ([]map[string]interface{}, error) {
+	resp, err := cr.callContentResolverApp(uuids, tid)
+	if err != nil {
+		return nil, fmt.Errorf("Error calling on url [%v] for content, error was: [%v]", cr.contentResolverAppURI, err.Error())
+	}
+	defer resp.Body.Close()
```

Comment:
ok, thx for the info. I assume this is not the case here since we are doing a `readAll`